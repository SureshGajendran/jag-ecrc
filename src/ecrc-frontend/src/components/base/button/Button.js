import React from "react";

export default function Button() {
  return <button>Nothing to do for now</button>;
}
