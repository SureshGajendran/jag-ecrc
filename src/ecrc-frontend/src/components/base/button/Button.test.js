import React from "react";
import { create } from "react-test-renderer";
import { Button } from "./Button";

describe("Button Component", () => {
  test("Matches the snapshot", () => {
    const buttonTest = {
      children: "test",
      onClick: () => jest.fn(),
      buttonStyle: "btn-warning",
      buttonSize: "btn-sm"
    };

    const button = create(<Button button={buttonTest} />);
    expect(button.toJSON()).toMatchSnapshot();
  });
});
