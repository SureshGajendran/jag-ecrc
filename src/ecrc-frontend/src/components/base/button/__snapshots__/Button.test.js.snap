// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Matches the snapshot 1`] = `
<button
  className="btn btn--primary--solid btn--medium"
  onClick={[Function]}
  type="button"
>
  test
</button>
`;
